tooling/minitest/src/lib.rs

```diff
@@ -57,8 +57,14 @@ pub fn assert_ub_eventually(prog: Program, attempts: usize, msg: &str) {
 }
 
 #[track_caller]
-pub fn assert_ill_formed(prog: Program) {
-    assert_eq!(run_program(prog), TerminationInfo::IllFormed);
+pub fn assert_ill_formed(prog: Program, msg: &str) {
+    let TerminationInfo::IllFormed(info) = run_program(prog) else {
+        panic!("program is not ill formed!")
+    };
+    if !(info.get_internal() == msg) {
+        panic!("{} vs {}", info.get_internal(), msg)
+    }
+    assert_eq!(info.get_internal(), msg);
```

Comment:
What is this about, why are you comparing twice?
```suggestion
    assert_eq!(info.get_internal(), msg, "program is ill-formed with a different error message");
```
`assert_eq!` will already print the two values if there is a discrepancy.

Reply:
You didn't quite apply the patch I suggested. Having a custom message is still good courtesy. :)

Reply:
ups, that was something I forgot to remove.